verticapy/plotting/base.py

```diff
@@ -69,7 +69,15 @@
     "summer": ["#728CA3", "#73C0F4", "#E6EFF3", "#F3E4C6", "#8F4F06"],
     "tropical": ["#7B8937", "#6B7436", "#F4D9C1", "#D72F01", "#F09E8C"],
     "india": ["#F1445B", "#65734B", "#94A453", "#D9C3B1", "#F03625"],
-    "default": ["#FE5016", "#263133", "#0073E7", "#FDE159", "#33C180", "#FF454F"],
+    "default": [
```

Comment:
please do not delete the old 'default' name it 'old'.

Reply:
done